tools/remote_build/include/test_config_common.bazelrc

```diff
@@ -0,0 +1,24 @@
+#@IgnoreInspection BashAddShebang
+# Copyright 2022 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# bazelrc for common configuration of gRPC tests
+
+build --verbose_failures=true
+
+# without verbose gRPC logs the test outputs are not very useful
+test --test_env=GRPC_VERBOSITY=debug
+
+# print output for tests that fail (default is "summary")
+build --test_output=errors
```

Comment:
Question: for passing cases, will their test logs be retained?

Reply:
yes, logs are kept for all test runs, this option only controls what get's printed to the console (= to the top level log in our case). The default "summary" options only prints whether a test failed or not but won't show the log for that test by default (which is less useful when on CI).